moon-store-mac-os/moon-store-mac-os/Helpers/AlertManager/AlertType.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AlertType.swift
+//  moon-store-mac-os
+//
+//  Created by Diana Zeledon on 4/12/24.
+//
+
+import AppKit
+
+enum AlertType {
+    case error(String)
```

Comment:
This associated value isn’t needed here. Always return the same value in messages

Reply:
If you look at the video it returns the message passed to the case.

Reply:
I know, but. Use an associated value for this doesn’t make a lot
Of sense

Reply:
Any suggestions for a better way?